pulsar-broker-common/src/main/java/org/apache/pulsar/broker/BaseGenerateDocumentation.java

```diff
@@ -90,7 +90,7 @@ protected String generateDocByFieldContext(String className, String type, String
         sb.append("|---|---|---|---|---|\n");
         for (Field field : fields) {
             FieldContext fieldContext = field.getAnnotation(FieldContext.class);
-            if (fieldContext == null) {
+            if (fieldContext == null || fieldContext.deprecated()) {
```

Comment:
@urfreespace Pls help check this change, shall we ignore the FileContext.deprecated param.

Reply:
@Anonymitaet any idea? I think it's not a tech question